docs/docsite/rst/user_guide/windows_faq.rst

```diff
@@ -3,4 +3,4 @@
 Windows Frequently Asked Questions
 ==================================
 
-This page has moved to :ref:`windows_faq`.
\ No newline at end of file
+This page has moved to :ref:`windows_faq`.
```

Comment:
This and all the windows changes here look like a bad rebase happened? The windows user guide is already merged in devel so shouldn't be showing up here....https://docs.ansible.com/ansible-core/devel/user_guide/windows_dsc.html

Reply:
Weird. Yeah I'll fix this. Thanks for catching it.